assets/js/scripts.js

```diff
@@ -182,19 +178,6 @@ jQuery( function ( $ ) {
 					$( this ).val( '' );
 				} );
 
-				// Set Country Dropdown.
```

Comment:
We need to replace this with something that effectively clears out radios and select inputs. When you switch to "Add New", all select inputs and radio inputs are pre-selected.

The state dropdown was originally being cleared out with that .html('') call, so something more generic for all select inputs would work for that. For radio inputs, we can clear it out by targetting the radio inputs with $('input[name={insert input name}]).attr('checked', false).

Reply:
Referring to this whole section that's been deleted (185 - 197)